README.md

```diff
@@ -87,6 +87,8 @@ Apt repository options for Kubernetes installation.
     kubernetes_yum_gpg_key:
       - https://packages.cloud.google.com/yum/doc/yum-key.gpg
       - https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+    kubernetes_gpg_check: true
+    kubernetes_repo_gpg_check: false
```

Comment:
I'd be willing to merge this if the defaults are kept as `true`. Someone can then override if need be, but the default should remain secure.